scripts/docker-build.sh

```diff
@@ -23,8 +23,8 @@ elif ! [ -r "$SSH_PRIVATE_KEY" ]; then
 fi
 
 
-DOCKER_BUILDKIT=1 docker buildx build --platform=linux/amd64 -f docker/Dockerfile${BUILD_TYPE} \
+DOCKER_BUILDKIT=1 docker buildx build --platform=linux/amd64 -f  docker/Dockerfile"${BUILD_TYPE}" \
```

Comment:
extra space after the `-f`?

Reply:
hmm shellcheck didn't tell me, moved now